spec/credit_transfer_transaction_spec.rb

```diff
@@ -71,4 +71,14 @@
       expect(SEPA::CreditTransferTransaction).not_to accept('', 'X' * 5, for: :category_purpose)
     end
   end
+
+  context 'Transaction Purpose: short message for creditor' do
+    it 'should allow valid value' do
```

Comment:
Perhaps we can avoid "should"ing in the language for it.

Example: "allows a valid value".

Reply:
Oh yes, it's something out of my diploma. I'll change it